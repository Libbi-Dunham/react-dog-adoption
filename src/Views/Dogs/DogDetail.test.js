import { render, screen } from '@testing-library/react';
import { MemoryRouter, Route } from 'react-router';
import Dog from './DogDetail.js';

test('renders the dog details', async () => {
  const { container } = render(
    <MemoryRouter initialEntries={['/dogs/1']}>
      <Route path="/dogs/:id" component={Dog} />
    </MemoryRouter>
  );
  await screen.findByText('Miklo');
  expect(container).toMatchSnapshot();
});
