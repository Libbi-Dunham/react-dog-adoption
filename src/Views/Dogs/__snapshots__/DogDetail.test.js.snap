// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the dog details 1`] = `
<div>
  <ul>
    <li
      style="list-style-type: none;"
    >
      <div
        class="dog"
      >
        <h2>
           
          Miklo
           
        </h2>
        <img
          src="https://placedog.net/1000/563?id=90"
        />
        <h2>
           
          Goofy and fun guy
           
        </h2>
        <a
          href="/dogs/1/DogEdit"
        >
          Edit
        </a>
      </div>
      <div>
        <button
          class="delete"
        >
          delete
        </button>
      </div>
    </li>
  </ul>
</div>
`;
